tox.ini

```diff
@@ -1,6 +1,21 @@
 [tox]
-envlist = py27,py37
```

Comment:
With drop Python 2 ?

Reply:
What is the interest of keeping it rather then the inability to use more recent python's features ?

Reply:
Done 👍  Thanks